data.tf

```diff
@@ -73,6 +73,7 @@ data template_file userdata {
     endpoint            = "${aws_eks_cluster.this.endpoint}"
     cluster_auth_base64 = "${aws_eks_cluster.this.certificate_authority.0.data}"
     max_pod_count       = "${lookup(local.max_pod_per_node, lookup(var.worker_groups[count.index], "instance_type", lookup(var.workers_group_defaults, "instance_type")))}"
+    pre_userdata        = "${lookup(var.worker_groups[count.index], "pre_userdata",lookup(var.workers_group_defaults, "pre_userdata"))}"
     additional_userdata = "${lookup(var.worker_groups[count.index], "additional_userdata",lookup(var.workers_group_defaults, "additional_userdata"))}"
```

Comment:
Just needing to get this in alignment wherever it appears.

Reply:
Will do. Would you prefer that we only change the internal additional_userdata to post_userdata or would you prefer to introduce a breaking change?

Reply:
cool I updated the changelog for now reflecting the `pre_userdata` variable so that it's captured.

Reply:
Thank you!